tests/Riverline/MultiPartParser/Converters/Commun.php

```diff
@@ -11,15 +11,17 @@
 
 namespace Riverline\MultiPartParser\Converters;
 
+use Riverline\MultiPartParser\StreamedPart;
+
 /**
  * Class Commun
  */
-class Commun extends \PHPUnit_Framework_TestCase
+abstract class Commun extends \PHPUnit_Framework_TestCase
```

Comment:
I have refactored this test by moving in the `Commun` class the `testParser` checks, reducing the code duplication as this test was copy/pasted on all converter implementations